cpp/ql/src/semmle/code/cpp/PrintAST.ql

```diff
@@ -7,3 +7,12 @@
 
 import cpp
 import PrintAST
+
+/**
+ * Temporarily tweak this class or make a copy to control which functions are
+ * printed.
+ */
+class Cfg extends PrintASTConfiguration {
+  /** Holds if the AST for `func` should be printed. */
```

Comment:
I would add `Tweak this predicate as needed`, perhaps in all caps.  Otherwise LGTM.